src/app/app.component.html

```diff
@@ -21,11 +21,9 @@
 
 
 <app-header></app-header>
-<app-breadcrumbs></app-breadcrumbs>
-<app-searchsection (emitSearchValue)="onSearchValue($event)"></app-searchsection>
-<app-courselist [searchValueResult]="searchValue"></app-courselist>
-<app-signin></app-signin>
-<app-courseview></app-courseview>
+<!--<app-pageview></app-pageview>-->
+<!--<app-signin></app-signin> -->
+<!--<app-courseview></app-courseview>-->
 <app-footer></app-footer>
 
 <router-outlet></router-outlet>
```

Comment:
`<router-outlet>` tag is used to contain matched component in your routes file (in your case for PageviewComponent/CourseviewComponent).

You just need to place it between your header and footer like that:

```
<app-header></app-header>
   <router-outlet></router-outlet>
<app-footer></app-footer>
```

Reply:
was fixed according to your comment (please find changes in the new commit)